tests/e2e/specs/App.spec.ts

```diff
@@ -0,0 +1,40 @@
+import Show from '@/models/Show';
+
+describe('App.vue', () => {
+  it('Menus and grapher exists', () => {
+    cy.visit('/');
+
+    cy.get('[data-test="app"]')
+      .find('.grapher');
+
+    cy.get('[data-test="app"]')
+      .find('.menu-top');
+
+    cy.get('[data-test="app"]')
+      .find('.menu-left');
+
+    cy.get('[data-test="app"]')
+      .find('.menu-right');
+
+    cy.get('[data-test="app"]')
+      .find('.menu-bottom');
+  });
+
+  it('window.app exists for e2e tests and can commit mutations', () => {
```

Comment:
this shouldnt be a test. only test full user workflows; e.g. user can create a show, click the save icon, reload the page, then see the same show data